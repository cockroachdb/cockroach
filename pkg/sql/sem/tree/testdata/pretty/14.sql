UPDATE abc SET (b, c) = (8, 9) WHERE a=2 and b=c ORDER BY v DESC LIMIT 1 RETURNING abc.b, c, 4 AS d
