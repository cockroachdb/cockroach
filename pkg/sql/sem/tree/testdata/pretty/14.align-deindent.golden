// Code generated by TestPretty. DO NOT EDIT.
// GENERATED FILE DO NOT EDIT
1:
-
UPDATE
	abc
SET
	(
		b,
		c
	)
		= (
			8,
			9
		)
WHERE
	a
	= 2
	AND b
		= c
ORDER BY
	v
		DESC
LIMIT
	1
RETURNING
	abc.b,
	c,
	4
		AS d

5:
-----
UPDATE
	abc
SET
	(
		b,
		c
	) = (
		8,
		9
	  )
WHERE
	a
	= 2
	AND b
		= c
ORDER BY
	v
		DESC
LIMIT
	1
RETURNING
	abc.b,
	c,
	4
		AS d

9:
---------
UPDATE
	abc
SET
	(
		b,
		c
	) = (
		8,
		9
	  )
WHERE
	a = 2
	AND b
		= c
ORDER BY
	v
		DESC
LIMIT
	1
RETURNING
	abc.b,
	c,
	4
		AS d

10:
----------
UPDATE
	abc
SET
	(
		b,
		c
	) = (
		8,
		9
	  )
WHERE
	a = 2
	AND b
		= c
ORDER BY
	v DESC
LIMIT
	1
RETURNING
	abc.b,
	c,
	4 AS d

13:
-------------
   UPDATE abc
      SET (
			b,
			c
          ) = (
				8,
				9
            )
    WHERE a
          = 2
      AND b
          = c
 ORDER BY v
			DESC
    LIMIT 1
RETURNING abc.b,
          c,
          4
			AS d

15:
---------------
   UPDATE abc
      SET (
			b,
			c
          ) = (
				8,
				9
            )
    WHERE a = 2
      AND b = c
 ORDER BY v
			DESC
    LIMIT 1
RETURNING abc.b,
          c,
          4
			AS d

16:
----------------
   UPDATE abc
      SET (
			b,
			c
          ) = (
				8,
				9
            )
    WHERE a = 2
      AND b = c
 ORDER BY v DESC
    LIMIT 1
RETURNING abc.b,
          c,
          4 AS d

20:
--------------------
   UPDATE abc
      SET (b, c) = (
					8,
					9
                 )
    WHERE a = 2
      AND b = c
 ORDER BY v DESC
    LIMIT 1
RETURNING abc.b,
          c,
          4 AS d

25:
-------------------------
   UPDATE abc
      SET (b, c) = (8, 9)
    WHERE a = 2 AND b = c
 ORDER BY v DESC
    LIMIT 1
RETURNING abc.b,
          c,
          4 AS d

26:
--------------------------
   UPDATE abc
      SET (b, c) = (8, 9)
    WHERE a = 2 AND b = c
 ORDER BY v DESC
    LIMIT 1
RETURNING abc.b, c, 4 AS d

103:
-------------------------------------------------------------------------------------------------------
UPDATE abc SET (b, c) = (8, 9) WHERE a = 2 AND b = c ORDER BY v DESC LIMIT 1 RETURNING abc.b, c, 4 AS d


